### DISCUSSION TITLE
Functionality for sharing notes

### DISCUSSION THREAD

**Discussion by blurredblu** [ostoaleonardo/notes#9] [Feature requests]
It would be great to have the ability to share note files through a few popular file formats (like rtf, docx, md, or pdf) and integrate it so that we can share a file through another app or as an attachment we can save to our phones.

I know that text can be selected and shared from the context menu, but being able to access the share functionality more easily and sharing a file itself would be a very useful feature.

**Comment by Discovir** [comment]
yes a feature like this would be great. like I was planning on using this app as my default journaling app but hesitated in case my phone gets broken or stolen, there's no backup so for now using Keep Notes. A cloud app integration would be nice as well.
